### PR TITLE
msh changes

### PR DESCRIPTION
**Pull request by mhisltd**
Resolves # . (This is applicable only if this pull request relates to a GitHub issue, delete the line otherwise)

Notes:
*
*
*

### CODE REVIEW

File: testme.bat [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+docker-compose down
+docker-compose -f docker-compose.yml -f docker-compose-test.yml pull
+docker-compose -f docker-compose.yml -f docker-compose-test.yml build
+docker-compose up -d --build
+#wget --retry-connrefused --tries=120 --waitretry=1 -O /dev/null http://localhost:8080/health
+echo "Waiting 30s for the docker to warm up" && timeout 30
+gradlew.bat clean test --info
+#docker-compose -f docker-compose.yml -f docker-compose-test.yml run -e GRADLE_OPTS tests
+start build/reports/tests/test/index.html
+open build/reports/tests/test/index.html
```
Comment (alectronic0) [inline testme.bat]: can remove open & xdg as these are for unix and now windows
also tests should now be `document-management-inter.... (check the docker-compose-test.yml)

File: src/test/groovy/uk/gov/hmcts/dm/it/ReadDocumentIT.groovy [hunk 2]
```diff
@@ -239,35 +239,102 @@ class ReadDocumentIT extends BaseIT {
...
+        def documentUrl = createDocumentAndGetUrlAs CITIZEN, ATTACHMENT_1, 'RESTRICTED', ['caseworker']
 
         givenRequest(CITIZEN_2)
                 .expect()
-                .statusCode(200)
+                .statusCode(403)
                 .when()
                 .get(documentUrl)
-
     }
```
Comment (alectronic0) [inline src/test/groovy/uk/gov/hmcts/dm/it/ReadDocumentIT.groovy:339]: If I am a user with no roles
I create a document with casework role + classification is private
then should I be able to get this file 

I need a test to prove this works please

**Review by alectronic0 — CHANGES_REQUESTED** [review]
please complete request

File: src/test/groovy/uk/gov/hmcts/dm/it/ReadDocumentIT.groovy (hunk 2 above)
Comment (mhisltd) [reply src/test/groovy/uk/gov/hmcts/dm/it/ReadDocumentIT.groovy:339]: Added the requested test in the integration tests

File: testme.bat (hunk 1 above)
Comment (mhisltd) [reply testme.bat]: Removed the requested lines and replaced the commented wget command with the Curl working command for windows.

**Comment by alectronic0** [comment]
I've updated your script try this on for size

**Review by alectronic0 — APPROVED** [review]

**Comment by mhisltd** [comment]
I've added some extra time to wait for the docker boxes to warmup.
- Fixed the issue of calling batches from testme.bat (They were initially not returning the control to the testme.bat after being executed.)

**Pull request merged**